models/zeroex/base/zeroex_base_api_fills.sql

```diff
@@ -0,0 +1,264 @@
+            logs.contract_address,
+            block_time                                      AS block_time,
+            bytearray_substring(DATA, 13, 20) AS maker,
+            0xdef1c0ded9bec7f1a1670819833240f027b25eff AS taker,
+            bytearray_substring(DATA, 45, 20) AS taker_token,
+            bytearray_substring(DATA, 77, 20) AS maker_token,
+            bytea2numeric(bytearray_substring(DATA, 110, 20)) AS taker_token_amount_raw,
+            bytea2numeric(bytearray_substring(DATA, 142, 20)) AS maker_token_amount_raw,
+            'BridgeFill'                                 AS type,
+            zeroex_tx.affiliate_address                     AS affiliate_address,
+            TRUE                                            AS swap_flag,
+            FALSE                                           AS matcha_limit_order_flag
+    FROM {{ source('base' ,'logs') }} logs
+    INNER JOIN zeroex_tx ON zeroex_tx.tx_hash = logs.tx_hash
+    WHERE topic0 = 0xe59e71a14fe90157eedc866c4f8c767d3943d6b6b2e8cd64dddcc92ab4c55af8
+        AND contract_address = 0xdb6f1920a889355780af7570773609bd8cb1f498
+
+        {% if is_incremental() %}
+        AND block_time >= date_trunc('day', now() - interval '7' day)
+        {% endif %}
+        {% if not is_incremental() %}
+        AND block_time >= cast('{{zeroex_v4_start_date}}' as date)
+        {% endif %}
+),
+/*
+direct_PLP AS (
+    SELECT 
+            plp.evt_tx_hash,
+            plp.evt_index               AS evt_index,
+            plp.contract_address,
+            plp.evt_block_time          AS block_time,f
+            provider                    AS maker,
+            recipient                   AS taker,
+            inputToken                  AS taker_token,
+            outputToken                 AS maker_token,
+            inputTokenAmount            AS taker_token_amount_raw,
+            outputTokenAmount           AS maker_token_amount_raw,
+            'LiquidityProviderSwap'     AS type,
+            zeroex_tx.affiliate_address AS affiliate_address,
+            TRUE                        AS swap_flag,
+            FALSE                       AS matcha_limit_order_flag
+    FROM {{ source('zeroex_base', 'ExchangeProxy_evt_LiquidityProviderSwap') }} plp
+    INNER JOIN zeroex_tx ON zeroex_tx.tx_hash = plp.evt_tx_hash
+
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc('day', now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    WHERE evt_block_time >= cast('{{zeroex_v3_start_date}}' as date)
+    {% endif %}
+), 
+ */
+all_tx AS (
+    
+    SELECT *
+    FROM ERC20BridgeTransfer
+    UNION ALL SELECT *
+    FROM BridgeFill
+    UNION ALL 
+    SELECT *
+    FROM NewBridgeFill 
+    
+    
+)
+
+SELECT 
+        all_tx.tx_hash,
+        all_tx.block_number,
+        all_tx.evt_index,
+        all_tx.contract_address,
+        all_tx.block_time,
+        try_cast(date_trunc('day', all_tx.block_time) AS date) AS block_date,
```

Comment:
while you're adding `block_month`, you can change from `try_cast` to `cast` for both

small thing we'll change universally post-migration